package.json

```diff
@@ -5,7 +5,7 @@
   "main": "bindings/node",
   "types": "bindings/node",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "tree-sitter test && crystal run test/stdlib_coverage",
```

Comment:
The convention for other TS grammars is to use `npm test` for testing the Node language bindings:
https://github.com/tree-sitter/tree-sitter-javascript/blob/108b2d4d17a04356a340aea809e4dd5b801eb40d/package.json#L58

So the coverage test doesn't belong here, but you could add a `test/run_all.sh` script and put it there if you want.